lib/w_common.dart

```diff
@@ -12,7 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export 'disposable.dart' show Disposable;
```

Comment:
should we deprecate this entry point now that we have browser- and VM-specific ones and this one is now technically exporting the VM version?